compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -61,6 +61,23 @@ ireeCodegenGetTunerRootOpsBinding(MlirModule module) {
   return ops;
 }
 
+static std::vector<int64_t> getIntArrayAttrValues(MlirAttribute attr) {
+  mlir::Attribute Attr = unwrap(attr);
+  auto arrayAttr = mlir::dyn_cast_or_null<mlir::ArrayAttr>(Attr);
```

Comment:
Use the C API for the type check instead -- notice how the whole file uses C APIs only.